tests/e2e/support/objects/app-files/utils/sidebar.ts

```diff
@@ -51,7 +51,7 @@ export const openPanel = async ({ page, name }: { page: Page; name: string }): P
   const nextPanel = page.locator(`#sidebar-panel-${name}-item`)
 
   await Promise.all([
-    locatorUtils.waitForEvent(nextPanel, 'focus'),
+    // locatorUtils.waitForEvent(nextPanel, 'focus'),
```

Comment:
what about this comment? does the row need to be deleted or does it need to be uncommented again?

Reply:
no it needs to be deleted